utils/zoneinfo64/src/rule.rs

```diff
@@ -91,6 +109,94 @@ pub(crate) enum RuleMode {
     DOW_LEQ_DOM,
 }
 
+/// The number of days in this year before this (0-based, from TzRuleDate.month) month starts
+fn days_before_month(m: u8, is_leap: bool) -> u16 {
+    const fn days_before_month_in_non_leap_year(month: u8) -> u16 {
+        (iso::const_fixed_from_iso(2021, month, 1).to_i64_date()
```

Comment:
you should still use `until` instead of `to_i64_date()`

Reply:
The polarity of `until` is backwards; I don't like that.

Reply:
it's a bug, change it